sos/report/plugins/ceph_mon.py

```diff
@@ -28,9 +28,9 @@ def setup(self):
         })
 
         self.add_copy_spec([
-            "/var/log/ceph/ceph-mon*.log",
-            "/var/lib/ceph/mon/",
```

Comment:
This is nullified due to the fact `/var/lib/ceph/mon/` is in the forbidden list as well. But there's not much else interesting there anyway besides `kv_backend`.